core/config/config.go

```diff
@@ -44,23 +43,23 @@ type Config struct {
 	GeneratorURL         string  `json:"generator_url"`
 	GeneratorAccessToken string  `json:"generator_access_token"`
 	ConfiguredAt         time.Time
-	BlockPub             string        `json:"block_pub"`
+	BlockPub             []byte        `json:"block_pub"`
 	Signers              []BlockSigner `json:"block_signer_urls"`
 	Quorum               int
 	MaxIssuanceWindow    time.Duration
 }
 
 type BlockSigner struct {
-	AccessToken string             `json:"access_token"`
-	Pubkey      chainjson.HexBytes `json:"pubkey"`
-	URL         string             `json:"url"`
+	AccessToken string `json:"access_token"`
+	Pubkey      []byte `json:"pubkey"`
```

Comment:
Until we have grpc, we should probably leave these as hex bytes. I think otherwise requests will have to use base64.